doc/cephadm/host-management.rst

```diff
@@ -197,6 +198,25 @@ This can be combined with service specifications (below) to create a cluster spe
 file to deploy a whole cluster in one command.  see ``cephadm bootstrap --apply-spec``
 also to do this during bootstrap. Cluster SSH Keys must be copied to hosts prior to adding them.
 
+Setting the initial CRUSH location of host
+==========================================
+
+Hosts can contain a ``location`` identifier which will instruct cephadm to 
+create a new CRUSH host located in the specified hirachy.
+
+.. code-block:: yaml
+
+    service_type: host1
+    hostname: node-00
+    addr: 192.168.0.10
+    location:
+      rack: rack1
+
+.. note:: 
+
+  The ``location`` attibute will be only affect the initial CRUSH location. Subsequent
+  changes of the ``location`` property will be ignored.
```

Comment:
If somebody was to accidentally put the wrong CRUSH location for the host and needed to change it, could they remove the host and add it back again, is there some better way or is it stuck like that? Either way, I think it would be nice to include that information here.

Reply:
I don't think this case is handled